resilient/resilient-annotation-processor/src/main/java/ru/tinkoff/kora/resilient/annotation/processor/aop/RetryableKoraAspect.java

```diff
@@ -62,18 +61,50 @@ public ApplyResult apply(ExecutableElement method, String superCall, AspectConte
         return new ApplyResult.MethodBody(body);
     }
 
-    private CodeBlock buildBodySync(ExecutableElement method, String superCall, String fieldRetry) {
-        final CodeBlock retrierExecution = CodeBlock.of("retry($L)", buildMethodCallable(method, superCall));
+    private static final ClassName CanRetryResult = ClassName.get("ru.tinkoff.kora.resilient.retry", "Retrier", "RetryState", "CanRetryResult");
+    private static final ClassName CanRetry = CanRetryResult.nestedClass("CanRetry");
+    private static final ClassName CantRetry = CanRetryResult.nestedClass("CantRetry");
+    private static final ClassName RetryExhausted = CanRetryResult.nestedClass("RetryExhausted");
+
 
+    private CodeBlock buildBodySync(ExecutableElement method, String superCall, String fieldRetry) {
+        var b = CodeBlock.builder()
+            .addStatement("var _cause = (Exception) null")
```

Comment:
Why generate 1-1 code from internalRetry when can be replaced with `retrier1.retry(() -> super.retrySyncVoid(arg));`?